Kernel/Heap/kmalloc.cpp

```diff
@@ -30,7 +30,7 @@ static constexpr size_t CHUNK_SIZE = 64;
 #endif
 
 #define POOL_SIZE (2 * MiB)
-#define ETERNAL_RANGE_SIZE (4 * MiB)
+#define ETERNAL_RANGE_SIZE (256 * KiB)
```

Comment:
Mention in the commit description what you observed (I think you said something about how we currently only need 31 KiB?)
That makes it obvious why you think that 256 KiB is a good idea.